.travis.yml

```diff
@@ -6,6 +6,8 @@ env:
   global:
    # Wheelhouse for pre-release wheels
    - PRE_WHEELS="https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com"
+   # Using a single thread can actually speed up some computations on Travis
+   - OPENBLAS_NUM_THREADS=1
```

Comment:
Won't have any effect --- it's using ATLAS (see below).

Reply:
... except perhaps for the OSX build.

Reply:
It should help when NumPy wheels are used, such as the `NUMPYSPEC="--pre --upgrade --timeout=60 -f $PRE_WHEELS numpy"` run, right? That one went from ~18 to ~13 minutes.

It looks like OSX and the other runs were unaffected.

Should we use OpenBLAS on some of these runs, or is it not done for some reason (e.g., to avoid OpenBLAS-related bugs)?